plugin/winresizer.vim

```diff
@@ -85,6 +92,7 @@ let g:winresizer_keycode_right = get(g:, 'winresizer_keycode_right', s:default_k
 let g:winresizer_keycode_finish = get(g:, 'winresizer_keycode_finish', s:default_keycode['finish'])
 let g:winresizer_keycode_cancel = get(g:, 'winresizer_keycode_cancel', s:default_keycode['cancel'])
 let g:winresizer_keycode_escape = get(g:, 'winresizer_keycode_escape', s:default_keycode['escape'])
+let g:winresizer_keycode_escape = get(g:, 'winresizer_keycode_enter', s:default_keycode['enter'])
```

Comment:
Please change `let g:winresizer_keycode_escape` to  `let g:winresizer_keycode_enter`.